### ISSUE TITLE
In synchronous mode, the action obtained by the get_control function has a delay of two time steps than the action of the apply_control function.

### ISSUE THREAD

**Issue by Roxanneoux** [carla-simulator/carla#4479]
### Environment:
CARLA 0.9.5
CARLA 0.9.6
CARLA 0.9.9

### Issue:
In synchronous mode, the action obtained by the get_control function has a delay of two time steps than the action applied by apply_control. However, the speed obtained by the get_velocity function has a delay of one time step than the speed applied by the set_velocity function. That is, it applies a control object on the next tick.
Is it a problem with the CARLA simulator?

Here is the code:
```
if action == 0:
        self.vehicle.apply_control(carla.VehicleControl(throttle=0.0, steer=0.0, brake=0.0))
elif action == 1:
        self.vehicle.apply_control(carla.VehicleControl(throttle=0.0, steer=0.0, brake=0.5))

self.world.tick()

c = self.vehicle.get_control() 
print('action:', action, 'c:', c)  
```   

I found that action and c do not correspond.

**Comment by Roxanneoux** [comment]
@corkyw10
Hi. Is this a problem with my code or a problem with the Carla simulator?

**Comment by DSantosO** [comment]
Hello @Roxanneoux,
This is a known issue that was fixed with the new synchronous pipeline in 0.9.11. You can check the details in the [release notes of this version.](http://carla.org/2020/12/22/release-0.9.11/)
